spark/setup_env

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [[ -z "${SPARK_HOME}" ]]
+then
+    echo 'Please set the SPARK_HOME variable to the path to a Spark installation, e.g. /opt/spark-2.0.2-bin-hadoop2.7' 
+>&2
+    exit -1
+fi
+
+spark_core_jars=( "${SPARK_HOME}/jars/spark-core*.jar" )
+if [ ${#spark_core_jars[@]} -eq 0 ]
+then
+    echo "Could not find a spark-core jar in ${SPARK_HOME}/jars, are you sure SPARK_HOME is set correctl?" >&2
+    exit -1
+fi
+
+export HAIL_HOME="$(dirname "$(cd "$(dirname $0)" && pwd)")"
```

Comment:
Is this assuming the current working directory is HAIL_HOME?

Reply:
No, $0 is the path to the script itself, so this dirname $0 is the directly containing the path (which is in bin), and dirname $(dirname $0) is the parent of that, which is what we want to set HAIL_HOME to.